awesome_cart/awc.py

```diff
@@ -1132,8 +1173,8 @@ def cart(data=None, action=None):
 			quotation.use_customer_fedex_account = 1 if data[0].get(
 				"address", {}).get("use_customer_fedex_account") else 0
 			quotation.flags.ignore_permissions = True
-			quotation.save()
-			frappe.db.commit()
+			#quotation.save()
+			#frappe.db.commit()
```

Comment:
Wouldn't this not save the "Use my Fedex account" value?

<sub>I've become wary of the octothorpe now.</sub>

Reply:
🐙

Reply:
If you notice, save and commit are already happening on line 1248. Why save and commit twice on the same call?

Reply:
But that save and commit is for when the `action` is "updateItem", whereas this one is for "calculate_shipping", and returns on line 1181. It'll give you the correct rates either way, but I'll confirm if the "Use Customer Fedex Account" check is properly applied.

Reply:
Ah sorry, was looking at the wrong thing. I meant 1179. So right before the return there is call to calc shipping and we are indeed saving and committing before returning from that.

Reply:
Ohh, I see. No problem, then. 👍😄